.github/CODEOWNERS

```diff
@@ -49,15 +44,18 @@
 /tests/modules/js-and-css/audit-enqueued-assets                             @manuelRod
 /tests/testdata/modules/js-and-css/audit-enqueued-assets                    @manuelRod
 
-# Module: Dominant Color Images
-/modules/images/dominant-color-images                                       @pbearne @spacedmonkey
-/tests/modules/images/dominant-color-images                                 @pbearne @spacedmonkey
-/tests/testdata/modules/images/dominant-color-images                        @pbearne @spacedmonkey
-
 # Plugin: Auto-sizes for Lazy-Loaded Images
 /plugins/auto-sizes                         @joemcgill
 /tests/plugins/auto-sizes                   @joemcgill
 
 # Plugin: Speculation Rules
 /plugins/speculation-rules                  @felixarntz
 /tests/plugins/speculation-rules            @felixarntz
+
+# Plugin: WebP Uploads
+/plugins/webp-uploads                       @adamsilverstein @felixarntz
+/tests/plugins/webp-uploads                 @adamsilverstein @felixarntz
+
+# Plugin: Dominant Color Images
+/plugins/dominant-color-images                                              @pbearne @spacedmonkey
+/tests/plugins/dominant-color-images                                        @pbearne @spacedmonkey
```

Comment:
Nit-pick, let's align the spacing with the other entries.

Reply:
Done https://github.com/WordPress/performance/pull/1011/commits/486014e1b6bd074ebbf110ef6a234b2248178c97